js_src/pid_display.js

```diff
@@ -88,9 +89,8 @@ app.MessageField.prototype.createDom = function() {
 app.MessageField.prototype.enterDocument = function() {
   app.MessageField.superClass_.enterDocument.call(this);
 
-  var tt = (
```

Comment:
I always thought this was an object at first glance, so I changed it to make it more clear that it's a string.

Reply:
I wondered if that was a linter fix, but perhaps not looking at the blame.